addons/website_slides/views/website_slides_templates_lesson_embed.xml

```diff
@@ -42,13 +42,13 @@
                                 t-attf-data-downloadable="#{False}"
                                 t-att-data-defaultpage="page">
                             <t t-if="is_embedded">
-                                <div id="slide_share" class="oe_slide_embed_option">
+                                <div id="slide_share" class="oe_slide_embed_option" style="display:none">
```

Comment:
Can you replace by `style="display: none;"`  to match how we usually write style attributes?

Otherwise LGTM.

Reply:
Roger that !

Reply:
I think you missed the semicolon at the end :(

Reply:
Oh I didn't see haha, I thought it was for the space
i'll copy paste it to be sure now haha :sweat_smile:  sorry about that